app-admin/etcd-wrapper/etcd-wrapper-3.1.6.ebuild

```diff
@@ -22,8 +22,15 @@ RDEPEND=">=app-emulation/rkt-1.9.1[rkt_stage1_fly]"
 S=${WORKDIR}
 
 src_install() {
+	local tag="v3.1.6"
```

Comment:
Since the ebuild has the same version now, maybe make this value `"v${PV}"` so updates only require a rename with no edits.